CMakeLists.txt

```diff
@@ -134,6 +135,13 @@ if(UR_ENABLE_TRACING)
         set_target_properties(xptifw PROPERTIES
             LIBRARY_OUTPUT_DIRECTORY ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}
         )
+
+        install(TARGETS xpti xptifw
```

Comment:
hm, I don't know how I feel about adding xpti and xptifw here. could this conflict with anything if a project that uses ur (e.g., SYCL) also has xpti/xptifw targets?
I think we should only do this (and the ur-common) if static linking is enabled.

Also, it may make sense  to do an alias:
```
add_library(ur_xpti ALIAS xpti)
add_library(ur_xptifw ALIAS xptifw)
```
so the names are different. But I'm not sure on that one.

Reply:
It seems that it's not possible to install targets from alias.
`CMake Error at CMakeLists.txt:143 (install): install TARGETS given target "ur_xpti" which is an alias.`